assets/css/amp-stories.css

```diff
@@ -3,3 +3,10 @@ amp-story-grid-layer[template="fill"] .wp-block-image { margin: 0; }
 .has-background {
 	padding: 10px !important;
 }
+
+.is-style-rounded {
+    border-radius: 5px;
+}
+.is-style-rounded img {
+	border-radius: 8px;
```

Comment:
@miina What's the reason here for the different radius? 8px vs. 5px.

Reply:
That's based on the designs, the images in template designs tend to have higher value for radius and are mostly larger as well. For example:
![Screen Shot 2019-04-03 at 6 37 35 PM](https://user-images.githubusercontent.com/3294597/55496542-a0aafe80-563f-11e9-85ad-1ceae863d1e8.png)

We could make it the same as well though, could be better for consistency. Thoughts?

Reply:
5px or 8px didn't really make a difference for me, so I'd rather unify them. I'll handle it.